server/service.go

```diff
@@ -0,0 +1,203 @@
+package server
```

Comment:
if this is an unused copy of service.go from the service package, it could be removed